samples/tfm/tfm_psa_template/src/main.c

```diff
@@ -203,20 +203,6 @@ void main(void)
 	if (err < 0) {
 		printk("Error mounting littlefs [%d]", err);
 	}
-
-	fs_mgmt_register_group();
-#endif
-#ifdef CONFIG_MCUMGR_CMD_OS_MGMT
-	os_mgmt_register_group();
-#endif
-#ifdef CONFIG_MCUMGR_CMD_IMG_MGMT
-	img_mgmt_register_group();
-#endif
-#ifdef CONFIG_MCUMGR_CMD_STAT_MGMT
-	stat_mgmt_register_group();
-#endif
-#ifdef CONFIG_MCUMGR_CMD_SHELL_MGMT
-	shell_mgmt_register_group();
 #endif
 #ifdef CONFIG_MCUMGR_SMP_BT
 	start_smp_bluetooth();
```

Comment:
@kapi-no ?

Reply:
This is symbol was once defined in the SMP Server sample:

```zephyr/samples/subsys/mgmt/mcumgr/smp_svr/src/bluetooth.c and bluetooth.h```

Currently, it is renamed to ```start_smp_bluetooth_adverts```. I guess this project was never built with the ```CONFIG_MCUMGR_SMP_BT``` configuration. Neither the header or implementation of the ```start_smp_bluetooth``` is added to the TFM sample in its Cmake. I am not planning to cover it within this PR

@joerchan, you might want to take a look at this

Reply:
Then it may be done under a separate task I think

Reply:
This would never have worked regardless, it is trying to call a function which is part of a completely different sample. Not really sure why it's in a tfm psa template sample anyway, issue can be raised for @joerchan